src/store/actions.js

```diff
@@ -0,0 +1,15 @@
+import { auth } from "../firebase";
+
+const actions = {
```

Comment:
Ok so we don't define each action as a const. Rather we define one variable called "actions" which contains a data object (?) with multiple functions defined in it